dataverse-webapp/src/main/webapp/resources/js/dv_rebind_bootstrap_ui.js

```diff
@@ -7,6 +7,24 @@
 function reinitializePrimefacesComponentsJS() {
     if (PrimeFaces.widget.SelectCheckboxMenu) {
         var originalSelectCheckboxMenuBindKeyEvents = PrimeFaces.widget.SelectCheckboxMenu.prototype.bindKeyEvents;
+        var originalSelectCheckboxMenuBindPanelKeyEvents = PrimeFaces.widget.SelectCheckboxMenu.prototype.bindPanelKeyEvents;
```

Comment:
Hmm isn't it good practice to use let instead of var?

Reply:
Just my 2 cents - using `let` will break IE8 compatibility, if that's something we care about

Reply:
I've heard of rule to use var only at the beginning of a function to avoid confusion with hoisting. But I don't see an issue to not use var altogether.

Reply:
After reading https://www.typescriptlang.org/docs/handbook/variable-declarations.html before I had to agree with the article. The var behavior is just more confusing for me.

Reply:
Well. I agree that let may be simpler to understand. But I'm not sure if we want to break compatibility when all the rest of js is compatible for couple statements. Maybe if it was new project then why not. You're pointing to typescript documentation where it is somewhat easier because you have transpiler that is able to replace let -> var with the right configuration.

Reply:
All right, fair point, let's leave it like that.